test/unit-test/FreeRTOS_Sockets/FreeRTOS_Sockets_utest.c

```diff
@@ -58,6 +59,72 @@
 
 #include "FreeRTOSIPConfig.h"
 
+extern List_t xBoundUDPSocketsList;
```

Comment:
This file is huge. It would probably be a good idea to split it up a bit.

Reply:
How do you propose we do that?

Reply:
Perhaps a file per function or set of functions?

Regardless, there are some config definitions that are mutually exclusive, so should probably be tested separately.

Reply:
Done - I split the tests into multiple files.
For the configuration, I am planning to add more tests later.